src/Components/nav/Nav.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import { NavLink } from 'react-router-dom';
+import './Nav.css';
+
+class Nav extends React.Component {
+  constructor(props) {
+    super(props);
+  }
```

Comment:
This seems to be purely a presentational component. Is the `constructor` really useful? Besides `eslint` seems to ask you the same question. Isn't it much simpler to just write:
```js
function Nav(){
   return 
}
```
instead of extending a base class, and returning inside `render` method with class based components? Looks like that is too much trouble already for a component which just displays UI without state.